WordPress/src/main/java/org/wordpress/android/ui/stats/service/StatsService.java

```diff
@@ -360,6 +369,54 @@ private boolean checkIfRequestShouldBeEnqueued(final RestClientUtils restClientU
         return true;
     }
 
+    // Call an updates on the installed widgets if the blog is the primary, the endpoint is Visits
+    // the timeframe is DAY or INSIGHTS, and the date = TODAY
+    private void updateWidgetsUI(String blogId, final StatsEndpointsEnum endpointName,
+                                 StatsTimeframe timeframe, String date, int pageRequested,
+                                 Serializable responseObjectModel) {
+        if (pageRequested != -1) {
```

Comment:
`pageRequested` is populated for a request when it's enqueued in the Stats service. The value of `-1` means that the parameter is missing from the request. Don't think we're "paging" on the VISITS endpoint yet, but better to have this check here.